feat: prima bozza qna (#201)
Il-Libro-Open-Source/book: qna.md

# Domande frequenti

> Qui si possono le risposte alle domande generali e più fretquenti sul progetto.

Reviewer: ```suggestion
Qui si possono trovare le risposte alle domande generali e più frequenti sul progetto.
```

---

Q: Di cosa parla il libro open source?
A: [Introduzione](https://github.com/Il-Libro-Open-Source/book/blob/main/docs/it/introduzione.md)

Q: Con che licenza è rilasciato?
A: Il libro è distribuito con una licenza "Attribution-NonCommercial-ShareAlike 4.0 International", maggiori informazioni [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/LICENSE.md)

Q: Su che mezzi sarà fruibile il libro?
A: Al momento è disponibile come file MD e pubblicato come GH pages: [https://il-libro-open-source.github.io/book/](https://il-libro-open-source.github.io/book/)

Q: Su che canali ufficiali posso seguire gli aggiornamenti sul libro oltre a GH?

> A: Su [linkedin](https://www.linkedin.com/company/il-libro-open-source) qui su [github](https://github.com/Il-Libro-Open-Source).

Reviewer: Qui propongo una delle due:

```markdown
Q: Su che canali ufficiali posso seguire gli aggiornamenti sul libro?
A: Su [linkedin](https://www.linkedin.com/company/il-libro-open-source) e qui su [github](https://github.com/Il-Libro-Open-Source).
```

oppure

```markdown
Q: Su che canali ufficiali posso seguire gli aggiornamenti sul libro oltre a GH?
A: Su [linkedin](https://www.linkedin.com/company/il-libro-open-source).
```

Q: In che lingua è disponibile?

> A: La prima stesura del libro sarà disponibile in italiano, in generale le issue, le PR e le discussioni si tengono in questa lingua; la struttura è stata pensata per includere altre traduzioni in futuro.

Reviewer: ```suggestion
A: La prima stesura del libro sarà disponibile in italiano; allo stesso modo le issue, le PR e le discussioni saranno in questa lingua; tuttavia la struttura è stata pensata per supportare, in futuro, altre traduzioni.
```

Q: Esiste un code of conduct?
A: Il code of conduct è disponibile [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md)

Q: Quale è la governance?

> A: Il libro viene mantenuto open e fruibile da tutti con una governance che vuole essere aperta, per questo da subito sono stati nominati una serie di figure dette Ambassador elette con voto su GH per gestire la crescita del libro, le posizioni vengono aperte temporalmente, questo periodo è in fase decisionale.

Reviewer 2: Qui aggiungerei link a https://github.com/Il-Libro-Open-Source/governance/blob/main/MEMBERSHIP.md

Author: Fatto

Q: Quali figure sono presenti?
A: Esistono due figure, l'Ambassador e il Contributor.

Q: Chi ne fa parte?
A: L'elenco è aggiornato qui: https://github.com/Il-Libro-Open-Source/governance/blob/main/AMBASSADOR.md

Q: In che maniera viene destituito un ambassador?
A: Un ambasssador può esser destituito con questi criteri: [https://github.com/Il-Libro-Open-Source/governance/blob/main/guidelines/ambassador-allontanamento.md](https://github.com/Il-Libro-Open-Source/governance/blob/main/guidelines/ambassador-allontanamento.md)

Q: Che ruolo svolge il contributor?

> A: Il contributor è una figura che collabora attivamente come un Ambassador ma non è referente per un particolare capitolo.

Reviewer 2: Prenderei parole quanto più vicine possibili al file di MEMBERSHIP

Q: Chi ne fa parte?
A: L'elenco è fruibile [qui](https://github.com/Il-Libro-Open-Source/governance/blob/main/CONTRIBUTOR.md)

Q: In che maniera viene eletto un contributor?
A: Il regolamento per il contributor è il medesimo degli ambassador.

Q: In che maniera viene destituito un contributor?
A: Il regolamento per il contributor è il medesimo degli ambassador.

> Q: Cosa sono i working group? e chi ne fa parte?

Reviewer: ```suggestion
Q: Cosa sono i Working Group? Chi ne fa parte?
```

> A: Sono gruppi di lavoro specializzati in vari aspetti come es. la revisine dei contenuti, trovi maggiori infomazioni [qui ](https://github.com/Il-Libro-Open-Source/governance/blob/main/WG.md)

Reviewer: ```suggestion
A: Sono gruppi di lavoro specializzati in vari aspetti come, ad esempio, la revisione dei contenuti; trovi maggiori informazioni [qui](https://github.com/Il-Libro-Open-Source/governance/blob/main/WG.md)
```

> Q: Come posso partecipare a uno di questi working group?

Reviewer 2: Questa la vedo più come domanda interna, non credo la porrei nelle FAQ..

Author: Eliminata

A: Ogni gruppo può decidere di allargare in caso di necessità il gruppo e apre le candidature per gli ambassador che sono interessanti, successivamente sarà creato un sondaggio.

Q: Mi sono accorto che alcuni contenuti non sono in linea col COC, come posso segnalarli?

> A: Nel COC è presente un link al form con il quale contattare direttamente il workgroup che analizzerà il caso.

Reviewer 2: Ribadirei il link al COC

Q: Quali sono le linee guida per scrivere contenuti?
A: Potete trovare un'introduzione [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES.md)

Q: Come posso contribuire?

> A: La contribuzione è aperta che si sia contributor, ambdassador o nessuno dei due, si può seguire la linea guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES-CONTENUTI.md)

Reviewer 2: Avendo mergiato stamattina, puoi linkare https://github.com/Il-Libro-Open-Source/book/blob/main/CONTRIBUTING.md

Q: Come si contribuisce? Con che mezzi?

> A: Attualmente si contribuisce tramite git lavorando sul repository principale come fosse codice.

Reviewer 2: Questa la vedo come domanda inglobata in quella sopra una volta che linki al CONTRIBUTING

Author: Fatto.

Q: Come posso proporre un argomento nuovo?

> A: Nelle discussioni è presente un topic sempre aperto per proposte e nuove idee: <<link>>

Reviewer 2: Nel CONTRIBUTING è tutto spiegato, direi di linkare lì alla sezione apposita

Q: Posso contribuire ad un capitolo già steso?

> A: Certo, basta seguire le linee guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES-CONTENUTI.md)

Reviewer 2: Idem come sopra

# Domande frequenti

Qui si possono trovare le risposte alle domande generali e più frequenti sul progetto.

---

### Di cosa parla il libro open source?

> > Puoi lggere l'introduzione [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/docs/it/introduzione.md).

Reviewer: ```suggestion
> Puoi leggere l'introduzione [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/docs/it/introduzione.md).
```

> > Puoi leggere l'introduzione [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/docs/it/introduzione.md).

Reviewer 2: ```suggestion
> Puoi leggere l'introduzione [qui](https://il-libro-open-source.github.io/book/).
```

### Con che licenza è rilasciato?

> Il libro è distribuito con una licenza "Attribution-NonCommercial-ShareAlike 4.0 International", maggiori informazioni [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/LICENSE.md)

### Su che mezzi sarà fruibile il libro?

> > Al momento è disponibile come file MD e pubblicato come GH pages: [https://il-libro-open-source.github.io/book/](https://il-libro-open-source.github.io/book/)

Reviewer 2: ```suggestion
> Al momento è disponibile come file MD e pubblicato su Github pages: [https://il-libro-open-source.github.io/book/](https://il-libro-open-source.github.io/book/)
```

### Su che canali ufficiali posso seguire gli aggiornamenti sul libro?

> > Su [linkedin](https://www.linkedin.com/company/il-libro-open-source) e qui su [github](https://github.com/Il-Libro-Open-Source).

Reviewer 2: ```suggestion
> Su [LinkedIn](https://www.linkedin.com/company/il-libro-open-source) e qui su [Github](https://github.com/Il-Libro-Open-Source).
```

### In che lingua è disponibile?

> La prima stesura del libro sarà disponibile in italiano; allo stesso modo le issue, le PR e le discussioni saranno in questa lingua; tuttavia la struttura è stata pensata per supportare, in futuro, altre traduzioni.

### Esiste un code of conduct?

> Il code of conduct è disponibile [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md)

### Quale è la governance?

> > Il libro viene mantenuto open e fruibile da tutti con una governance che vuole essere aperta, per questo da subito sono stati nominati una serie di figure dette Ambassador elette con voto su GH per gestire la crescita del libro, le posizioni vengono aperte temporalmente, questo periodo è in fase decisionale. Maggiori informazioni [qui](https://github.com/Il-Libro-Open-Source/governance/blob/main/MEMBERSHIP.md)

Reviewer: ```suggestion
> Il libro viene mantenuto open e fruibile da tutti con una governance che vuole essere aperta, per questo da subito sono stati nominati una serie di figure dette Ambassador elette con voto su Github per gestire la crescita del libro, le posizioni vengono aperte temporalmente, questo periodo è in fase decisionale. Maggiori informazioni [qui](https://github.com/Il-Libro-Open-Source/governance/blob/main/MEMBERSHIP.md)
```

### Quali figure sono presenti?

> Esistono due figure, l'Ambassador e il Contributor.

### Chi ne fa parte?

> L'elenco è aggiornato qui: https://github.com/Il-Libro-Open-Source/governance/blob/main/AMBASSADOR.md

### In che maniera viene destituito un ambassador?

> > Un ambasssador può esser destituito con questi criteri: [https://github.com/Il-Libro-Open-Source/governance/blob/main/guidelines/ambassador-allontanamento.md](https://github.com/Il-Libro-Open-Source/governance/blob/main/guidelines/ambassador-allontanamento.md)

Reviewer: ```suggestion
> Un ambassador può esser destituito con questi criteri: [https://github.com/Il-Libro-Open-Source/governance/blob/main/guidelines/ambassador-allontanamento.md](https://github.com/Il-Libro-Open-Source/governance/blob/main/guidelines/ambassador-allontanamento.md)
```

> I Contributors sono membri chiave della community che contribuiscono attivamente al progetto e alle discussioni, seguendo le linee guida stabilite. Devono rimanere attivi e soddisfare alcuni requisiti. In cambio, hanno accesso privilegiato all'organizzazione del progetto, possono assistere altri membri e hanno l'opportunità di avanzare al ruolo di Ambassador. Per maggiori informazioni leggi [qui](https://github.com/Il-Libro-Open-Source/governance/blob/main/MEMBERSHIP.md)

### Chi ne fa parte?

> L'elenco è fruibile [qui](https://github.com/Il-Libro-Open-Source/governance/blob/main/CONTRIBUTOR.md)

### In che maniera viene eletto un contributor?

> Il regolamento per il contributor è il medesimo degli ambassador.

### In che maniera viene destituito un contributor?

> Il regolamento per il contributor è il medesimo degli ambassador.

### Cosa sono i Working Group? Chi ne fa parte?

> Sono gruppi di lavoro specializzati in vari aspetti come, ad esempio, la revisione dei contenuti; trovi maggiori informazioni [qui](https://github.com/Il-Libro-Open-Source/governance/blob/main/WG.md)

### Mi sono accorto che alcuni contenuti non sono in linea col COC, come posso segnalarli?

> > Nel COC è presente un link al form con il quale contattare direttamente il workgroup che analizzerà il caso; trovi maggiorni informazioni [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md).

Reviewer: ```suggestion
> Nel COC è presente un link al form con il quale contattare direttamente il working group che analizzerà il caso; trovi maggiori informazioni [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md).
```

### Quali sono le linee guida per scrivere contenuti?

> Potete trovare un'introduzione [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES.md)

### Come posso contribuire?

> > La contribuzione è aperta sia ai Contributors che agli Ambdassadors; si può seguire la linea guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES-CONTENUTI.md)

Reviewer 2: ```suggestion
> La contribuzione è aperta a chiunque, non solo a Contributor e Ambdassador; si può seguire la linea guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES-CONTENUTI.md)
```

> > La contribuzione è aperta che si sia contributor, ambdassador o nessuno dei due, si può seguire la linea guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES-CONTENUTI.md)

Reviewer: ```suggestion
> La contribuzione è aperta sia ai Contributors che agli Ambdassadors; si può seguire la linea guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES-CONTENUTI.md)
```

### Come si contribuisce? Con che mezzi?

> > Attualmente si contribuisce tramite git lavorando sul repository principale come fosse codice, trovi una guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/CONTRIBUTING.md).

Reviewer 2: ```suggestion
> Attualmente si contribuisce direttamente tramite git lavorando sul repository come fosse codice, trovi una guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/CONTRIBUTING.md).
```

### Posso contribuire ad un capitolo già steso?

> Certo, basta seguire le linee guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/CONTRIBUTING.md)

### Come posso far approvare un contenuto?

> > Una volta creata la PR verrà presa in esame per controllare l'aderenza a tutte le guidelines e coc, successivamente servono tre ambassador per l'approvazione della PR, potete trovare tutti i dettagli [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES-CONTENUTI.md)

Reviewer 2: ```suggestion
> Una volta creata la PR verrà presa in esame per controllare l'aderenza a tutte le guidelines e coc, successivamente un certo numero di ambassador dovrà approvare la PR. Potete trovare tutti i dettagli [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES-CONTENUTI.md)
```
